amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -325,6 +342,19 @@ const ColumnList: React.FC<ColumnListProps> = ({
     ];
   }
 
+  const openedColumnsMap = {};
+  const handleRowExpand = (rowValues) => {
+    if (openedColumnsMap[rowValues.name]) return;
```

Comment:
We usually add curly braces here, just as a habit. I wasn't expected an inline return when I read this.